requirements/min.old

```diff
@@ -31,7 +31,8 @@ pandas==0.23.2
 pycocotools==2.0.0
 radiant-mlhub==0.2.1
 rarfile==3.0
-scipy==1.2.0
+scikit-image==0.18.0
+scipy==1.7.0
```

Comment:
Why the scipy version bump? You'll also need to bump this lower bound in other places

Reply:
The older versions kept throwing a  `AttributeError:'scipy.signal._max_len_seq_inner.array' has no attribute '__reduce_cython__'`.

Reply:
Locally or in CI?

Reply:
CI